distro.py

```diff
@@ -1034,6 +1041,8 @@ def _parse_distro_release_content(line):
                 distro_info['version_id'] = m.group(2)[::-1]
             if m.group(1):
                 distro_info['codename'] = m.group(1)[::-1]
+        elif line:
+            distro_info['name'] = line.strip()
```

Comment:
it still can be changed if it produces errors. i think simplicity beats purity here.